### PR TITLE
Use G_LIST in dist/threads

### PR DESCRIPTION
**Pull request by leonerd**

### CODE REVIEW

**Comment by khwilliamson** [comment]
ppport needs releasing before this can be merged

**Comment by leonerd** [comment]
Oh I thought I checked that `threads` doesn't actually use ppport.h; it was one of the only dists that doesn't

File: dist/threads/threads.xs [hunk 1]
```diff
@@ -46,6 +46,10 @@
 # define CLANG_DIAG_RESTORE_DECL CLANG_DIAG_RESTORE dNOOP
 #endif
 
+#ifndef G_LIST
```
Comment (khwilliamson) [inline dist/threads/threads.xs]: Remove these lines, and merge this PR

**Comment by khwilliamson** [comment]
Devel::PPPort is no on CPAN with G_LIST.  This PR can now be merged, but after the changes I requested

**Comment by Leont** [comment]
@jdhedden ^

File: dist/threads/threads.xs (hunk 1 above)
Comment (leonerd) [reply dist/threads/threads.xs]: Now done by 90e743c0ed

**Pull request merged**